subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/finalizer/verdict/Verdict.kt

```diff
@@ -19,7 +19,7 @@ internal sealed class Verdict {
 
     data class Failure(
         override val testResults: Collection<AndroidTest>,
-        val lostTests: Collection<AndroidTest.Lost>,
-        val failedTests: Collection<TestStaticData>
+        val notReportedTests: Collection<AndroidTest.Lost>,
+        val unsuppressedFailedTests: Collection<TestStaticData>
```

Comment:
> unsuppressed

Should and can we know at this level the reason of failure?

Reply:
No need for now; If we will need it in future it's possible to pass whole `AndroidTest` object

Reply:
I mean that on this level of abstraction it looks like `failed tests`. 
For clients of the verdict, these tests are the reason of failure. It's impossible to say here why they failed. Thus, knowledge about "suppressing" here is excessive, at least.